### PR TITLE
feat: allow specifying an initial value for a prompt

### PR DESCRIPTION
**Pull request by MatanRubin**
This change allows the user to specify an initial value for a prompt, similar to how "select" prompts allow specifying the initial position of the cursor.
If an initial value is given, it is automatically placed in the prompt, and the cursor is moved to the end, as if the user just typed this input himself.

### CODE REVIEW

File: beaupy/_beaupy.py [hunk 1]
```diff
@@ -96,8 +98,8 @@ def prompt(
     """
     rendered = ''
     with _cursor_hidden(console), Live(rendered, console=console, auto_refresh=False, transient=True) as live:
-        value: List[str] = []
-        cursor_index = 0
+        value: List[str] = [initial_value] if initial_value else []
```
Comment (petereon) [inline beaupy/_beaupy.py:101]: There is a minor bug on this line, this needs to be:
```python
[*initial_value] if initial_value else []
```
rather than
```python
[initial_value] if initial_value else []
```

Otherwise the application raises `IndexError: list assignment index out of range` when I start a form with `initial_value='Hello, '` and press backspace.

Comment (MatanRubin) [reply beaupy/_beaupy.py]: Great catch, thanks! I fixed the bug and added some more tests for edge cases.

Comment (petereon) [reply beaupy/_beaupy.py]: I thank you. I am going to merge this and do a release today.
